docs/advanced.md

```diff
@@ -0,0 +1,91 @@
+# Advanced Options and Configuration
+
+This section contains advanced information describing the different ways you can run and manage RKE2:
+
+- [Certificate rotation](#certificate-rotation)
+- [Auto-deploying manifests](#auto-deploying-manifests)
+- [Configuring containerd](#configuring-containerd)
+- [Secrets Encryption Config](#secrets-encryption-config-experimental)
+- [Node labels and taints](#node-labels-and-taints)
+- [Starting the server with the installation script](#starting-the-server-with-the-installation-script)
+
+# Certificate Rotation
+
+By default, certificates in RKE2 expire in 12 months.
+
+If the certificates are expired or have fewer than 90 days remaining before they expire, the certificates are rotated when RKE2 is restarted.
+
+# Auto-Deploying Manifests
+
+Any file found in `/var/lib/rancher/rke2/server/manifests` will automatically be deployed to Kubernetes in a manner similar to `kubectl apply`.
+
+For information about deploying Helm charts using the manifests directory, refer to the section about [Helm.](helm.md)
+
+# Configuring containerd
+
+RKE2 will generate config.toml for containerd in `/var/lib/rancher/rke2/agent/etc/containerd/config.toml`.
+
+For advanced customization for this file you can create another file called `config.toml.tmpl` in the same directory and it will be used instead.
+
+The `config.toml.tmpl` will be treated as a Go template file, and the `config.Node` structure is being passed to the template. [This template](https://github.com/rancher/k3s/blob/master/pkg/agent/templates/templates.go#L16-L32) example on how to use the structure to customize the configuration file.
+
+# Secrets Encryption Config
+RKE2 supports the secrets encryption at rest, and will do the following automatically:
```

Comment:
Am I reading this wrong or is the first "the" weird?

Reply:
it is. I copied this from k3s and didn't proofread the english as much as I probably could have, other than to remove stuff that's not relevant to rke2.

Reply:
@cjellick should we fix and backport to k3s or disregard for now?